src/api-reference/user-provisioning/spend/v4.spend-user-retrieval.md

```diff
@@ -36,82 +55,83 @@ layout: reference
 ```shell
 GET https://www.us.api.concursolutions.com/spend/v4/Users
 ```
+
 #### Parameters
 
-|Name|Type|Format|Description|
-|---|---|---|---|
-|`startIndex`|`integer`|-|The starting index of the paginated result (indexes from 1).|
-|`itemsPerPage`|`integer`|-|The number of user resources to return on a single page. Maximum: 100|
-|`filter`|`string`|-|The SCIM compliant filter string to be used when retrieving user resources.|
+| Name           | Type      | Format | Description                                                                 |
+| -------------- | --------- | ------ | --------------------------------------------------------------------------- |
+| `startIndex`   | `integer` | -      | The starting index of the paginated result (indexes from 1).                |
+| `itemsPerPage` | `integer` | -      | The number of user resources to return on a single page. Maximum: 100       |
+| `filter`       | `string`  | -      | The SCIM compliant filter string to be used when retrieving user resources. |
 
 **Note:** Currently not all aspects of SCIM filtering are supported. As of right now the following fields are implemented with the corresponding set of operators:
 
-* `cashAdvanceAccountCode` { **eq**, **ne** }
-* `country` { **eq**, **ne** }
-* `customData` { **complexValue** that contains **and**, **eq** }
-* `ledgerCode` { **eq**, **ne** }
-* `locale` { **eq**, **ne** }
-* `nonEmployee` { **eq**, **ne** }
-* `reimbursementCurrency` { **eq**, **ne** }
-* `reimbursementType` { **eq**, **ne** }
-* `stateProvince` { **eq**, **ne** }
-* `testEmployee` { **eq**, **ne** }
+- `cashAdvanceAccountCode` { **eq**, **ne** }
```

Comment:
Would it be easier to read if this is in table format?
Sample:
| Param    | EQ  | NE |
|:---------|:---:|:---:|
| country   |  X  |  X  |

Reply:
Maybe, im fine either way. We can see what clients think too.